packages/lending/package.json

```diff
@@ -24,9 +24,9 @@
   },
   "dependencies": {
     "@aave/math-utils": "^1.21.0",
-    "@protocolink/api": "^1.0.21",
+    "@protocolink/api": "^1.0.25",
     "@protocolink/common": "^0.3.10",
-    "@protocolink/core": "^0.4.11",
+    "@protocolink/core": "^0.4.12",
     "@protocolink/logics": "^1.1.7",
```

Comment:
Upgrade to v.1.1.1 so that the `yarn.lock` only contains one '@protocolink/logics'